### DISCUSSION TITLE
无法移动哗哩哗哩UWP

### DISCUSSION THREAD

**Discussion by AvoidGuest** [ywmoyue/biliuwp-lite#753] [Q&A] [answered]
我习惯把软件都装在D盘里UWP应用也不例外，其他UWP应用都可以移动到D盘但哗哩哗哩UWP是个例外请问是怎么回事？
![image](https://github.com/user-attachments/assets/e3f09912-8862-403e-a033-be1bf014eb0f)
![image](https://github.com/user-attachments/assets/9ec671ca-4437-45f7-85ce-51ea4b0f5f77)

**Comment by ywmoyue** [comment, marked as answer]
参考微软文档
> https://learn.microsoft.com/en-us/powershell/module/appx/move-appxpackage?view=windowsserver2022-ps#example-1-move-a-package-to-a-volume-specified-by-a-path

> https://learn.microsoft.com/en-us/powershell/module/appx/add-appxpackage?view=windowsserver2022-ps#-volume

我测了下可以卸载应用后修改脚本`Add-AppDevPackage.ps1`在`Add-AppxPackage -Path $DeveloperPackagePath.FullName -DependencyPath $DependencyPackages.FullName -ForceApplicationShutdown`和`Add-AppxPackage -Path $DeveloperPackagePath.FullName -ForceApplicationShutdown`之后添加参数` -Volume D:\`

然后运行`Install.ps1`就能安装到D盘了

![image](https://github.com/user-attachments/assets/d4efb905-6703-4584-8101-f10a471d01cd)
